packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -92,18 +95,24 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 			const blockName = getBlockName( clientId );
 			const rootClientId = getBlockRootClientId( clientId );
 			const blockType = getBlockType( blockName );
+			const lightBlockWrapper =
+				blockType.apiVersion > 1 ||
+				hasBlockSupport( blockType, 'lightBlockWrapper', false );
+
 			return {
 				index: getBlockIndex( clientId, rootClientId ),
 				mode: getBlockMode( clientId ),
 				name: blockName,
-				blockAPIVersion: blockType.apiVersion,
 				blockTitle: blockType.title,
 				isPartOfSelection: isSelected || isPartOfMultiSelection,
 				adjustScrolling:
 					isSelected || isFirstMultiSelectedBlock( clientId ),
 				enableAnimation:
 					! isTyping() &&
 					getGlobalBlockCount() <= BLOCK_ANIMATION_THRESHOLD,
+				hasBlockAPIVersionWarning:
+					clientId === blockEditContext.clientId &&
```

Comment:
I think `blockEditContext.clientId` has to be included in the dependency array?

Reply:
Good catch 👍🏻